blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_3_materials/formula_3_23.py

```diff
@@ -0,0 +1,48 @@
+"""Formula 3.23 from NEN-EN 1992-1-1+C2:2011: Chapter 3 - Materials."""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import MM, MPA
+
+
+class Form3Dot23TensileBendingStrength(Formula):
+    """Class representing formula 3.23 for the calculation of the average tensile bending strength."""
+
+    label = "3.23"
+    source_document = NEN_EN_1992_1_1_C2_2011
+
+    def __init__(
+        self,
+        h: MM,
+        f_ctm: MPA,
+    ) -> None:
+        """[fctm,fl] Average tensile bending strength [MPa].
+
+        NEN-EN 1992-1-1+C2:2011 art.3.1.8(1) - Formula (3.23)
+
+        Parameters
+        ----------
+        h : MM
+            [h] Total element height [mm].
+        f_ctm : MPA
+            [fctm] Average axial tensile strength following table 3.1 [MPa].
+
+        Returns
+        -------
+        None
+        """
+        super().__init__()
+        self.h = h
+        self.f_ctm = f_ctm
+
+    @staticmethod
+    def _evaluate(
+        h: MM,
+        f_ctm: MPA,
+    ) -> float:
+        """Evaluates the formula, for more information see the __init__ method"""
```

Comment:
This is fine, but the __init__ says nothing about the return, since the __init__ itself returns None. Return docstring should be added here.

Reply:
We have opted for brief docstrings for the _evaluate method, considering its status as a private function